backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT22.json

```diff
@@ -20,7 +20,8 @@
     "Register": false,
     "KeyVault": false,
     "SystemRegister": false,
-    "SystemUser": false
+    "SystemUser": false,
+    "SystemUserClientDelegation": false
```

Comment:
Should this be added to the standard appsettings also to make sure all other environments are set to false?